packages/lesswrong/components/ea-forum/EAHome.tsx

```diff
@@ -13,12 +13,39 @@ const showHandbookBannerSetting = new DatabasePublicSetting<boolean>('showHandbo
 const showEventBannerSetting = new DatabasePublicSetting<boolean>('showEventBanner', false)
 const showMaintenanceBannerSetting = new DatabasePublicSetting<boolean>('showMaintenanceBanner', false)
 
+const getStructuredData = () => ({
+  "@context": "http://schema.org",
+  "@type": "WebSite",
+  "url": `${getSiteUrl()}`,
+  "potentialAction": {
+    "@type": "SearchAction",
+    "target": `${getSiteUrl()}search?query={search_term_string}`,
+    "query-input": "required name=search_term_string"
+  },
+  "mainEntityOfPage": {
+    "@type": "WebPage",
+    "@id": `${getSiteUrl()}`,
+  },
+  ...(isEAForum && {
+    "description": [
+      "A forum for discussions and updates on effective altruism. Topics covered include",
+      "global health, AI safety, biosecurity, animal welfare, philosophy, policy, forecasting,",
+      "and effective giving. Users can explore new posts, engage with the community,",
+      "participate in recent discussions, and discover topics, events,",
+      "and groups. An accessible space for sharing and learning about approaches to tackling",
+      "the world's most pressing problems."
```

Comment:
Apparently it won't directly include this description in the search results so it's fine to stuff in keywords a bit more